examples/nlp/scripts/get_wkt2.sh

```diff
@@ -7,14 +7,15 @@ Copyright by the AWD LSTM authors.
 
 echo "- Downloading WikiText-2"
 
+cd /raid
 wget --continue -P data/lm/ https://s3.amazonaws.com/research.metamind.io/wikitext/wikitext-2-v1.zip
 unzip -q data/lm/wikitext-2-v1.zip -d data/lm
-cd data/lm/wikitext-2
+cd /raid/data/lm/wikitext-2
```

Comment:
this only works if users have /raid on their machine

Reply:
Fixed, thank you